crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,31 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
```

Comment:
@rhafer what do you think about @MaximilianMeister suggestion to change the name of the method?